src/main/kotlin/za/ac/sun/plume/drivers/OverflowDbDriver.kt

```diff
@@ -0,0 +1,255 @@
+                    convertDispatchType(v.dispatchType()), v.typeFullName(),
+                    getOrElse(v.dynamicTypeHintFullName().headOption(), ""), v.name(), v.signature(), v.code(), v.order(),
+                    getOrElse(v.lineNumber(), 0), getOrElse(v.columnNumber(), 0)
+            )
+            is File -> FileVertex(v.name(), v.order())
+            is Identifier -> IdentifierVertex(v.name(), v.typeFullName(), v.code(), v.order(), v.argumentIndex(), getOrElse(v.lineNumber(), 0), getOrElse(v.columnNumber(), 0))
+            is MetaData -> MetaDataVertex(v.language(), v.version())
+            is Method -> MethodVertex(v.name(), v.fullName(), v.signature(), v.code(),
+                    getOrElse(v.lineNumber(), 0), getOrElse(v.columnNumber(), 0), v.order())
+            is MethodParameterIn -> MethodParameterInVertex(v.code(), convertEvalStrategy(v.evaluationStrategy()) , v.typeFullName(), getOrElse(v.lineNumber(), 0), v.name(), v.order())
+            is MethodReturn -> MethodReturnVertex(v.typeFullName(),
+                    convertEvalStrategy(v.evaluationStrategy()), v.code(),
+                    getOrElse(v.lineNumber(), 0), getOrElse(v.columnNumber(),0), v.order())
+            is Literal -> LiteralVertex(v.typeFullName(), v.code(), v.order(), v.argumentIndex(), getOrElse(v.lineNumber(), 0), getOrElse(v.columnNumber(), 0))
+            is Local -> LocalVertex(v.code(), v.typeFullName(), getOrElse(v.lineNumber(), 0), getOrElse(v.columnNumber(), 0), v.name(), v.order())
+            is NamespaceBlock -> NamespaceBlockVertex(v.name(), v.fullName(), v.order())
+            is Return -> ReturnVertex(getOrElse(v.lineNumber(), 0), getOrElse(v.columnNumber(), 0), v.order(), v.argumentIndex(), v.code())
+            is TypeDecl -> TypeDeclVertex(v.name(), v.fullName(), v.astParentFullName(), v.order())
+            else -> {
+                println(v)
+                TODO("Not implemented")
+            }
+        }
+    }
+
+    private fun convertEvalStrategy(str : String) : EvaluationStrategy {
+        return EvaluationStrategy.valueOf(str)
+    }
+
+    private fun convertDispatchType(str : String) : DispatchType {
+        return DispatchType.valueOf(str)
+    }
+
+    private fun <T>getOrElse(opt : scala.Option<T>, def : T) : T {
+        class F<T>(val x : T) : AbstractFunction0<T>() {
+            override fun apply(): T {
+                return x;
+            }
+        }
+        return opt.getOrElse(F(def))
+    }
+
+    override fun exists(v: PlumeVertex): Boolean {
+        return (graph.node(v.hashCode().toLong()) != null)
+    }
+
+    override fun exists(fromV: PlumeVertex, toV: PlumeVertex, edge: EdgeLabel): Boolean {
+        val srcNode = graph.node(fromV.hashCode().toLong())
+        val dstNode = graph.node(toV.hashCode().toLong())
+        return srcNode != null && srcNode.out(edge.name).asSequence().toList().filter { node -> node.id().equals(dstNode.id()) }.isNotEmpty()
```

Comment:
You won't need to check for null if you use a null-safe `let` and combine it with the elvis operator:
```kotlin
graph.node(fromV.hashCode().toLong())?.let { srcNode ->
   val dstNode = graph.node(toV.hashCode().toLong())
   return srcNode.out(edge.name).asSequence().toList()
               .any { node -> node.id() == dstNode.id() }
} ?: return false
```

Reply:
Done